SWLOR.Game.Server/Data/Migrations/2019-09-23.2.sql

```diff
@@ -0,0 +1,85 @@
+﻿INSERT INTO dbo.Perk (ID , Name , IsActive , BaseCastingTime , Description , PerkCategoryID , CooldownCategoryID , ExecutionTypeID ,IsTargetSelfOnly ,Enmity , EnmityAdjustmentRuleID , CastAnimationID , ForceBalanceTypeID)
+VALUES	(88 ,
+		N'Speedy Crafting' ,
+		1 ,
+		0 ,
+		N'Reduces the amount of time it takes to Craft. This does not stack with other Speedy Perks' ,
```

Comment:
Reword "Craft" to read "craft items" for me. I don't see a reason to keep the individual speedy perks if this one supersedes those, so get rid of the stacking phrasing too.

Reply:
I kept the original ones because this line should be more expensive and I don't feel ti is fair to remove what we currently have  to to replace it with a more costly option for the players that only do 1 craft. This line is supposed to help the dedicated crafters that pick up multiple lines.
I can still remove the old ones i just think it would be better to keep them and add some better documentation so players know get one or the other, not both.

Reply:
Can we just lower the SP cost for the universal speedy perk then? They honestly just speed up the crafting process so I'm really fine with everyone picking it up for the same amount that an individual one costs now.

Reply:
Sure thing.